build.gradle

```diff
@@ -22,7 +22,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:2.9.0'
     compile 'org.apache.oltu.oauth2:org.apache.oltu.oauth2.client:1.0.2'
-    compile 'com.microsoft.graph:microsoft-graph-core:0.1.0-SNAPSHOT'
+    compile 'com.microsoft.graph:microsoft-graph-core:1.0.0'
```

Comment:
@NakulSabharwal could you please let me know when will this PR be merged? We are facing a similar issue, could you please help us by giving the official JAR and also let us know by when it will be available as part of Maven import ?
Thanks in Advance